packages/client/hmi-client/src/components/models/tera-model-diagram.vue

```diff
@@ -623,6 +624,9 @@ const prepareTransitionEdit = () => {
 const addNode = async () => {
 	if (!renderer) return;
 	const node = editNodeObj.value;
+	if (!node?.id) {
+		return;
+	}
 	if (props.model?.model.states.find((s) => s.id === node.id)) {
```

Comment:
ive added this since the button is disabled when editNodeObj.id is empty, so i have added this check for the enter action to capture it